core/src/common/lwm2m_debug.c

```diff
@@ -98,3 +98,33 @@ void Lwm2m_SetLogLevel(DebugLevel level)
 {
     debugLevel = level;
 }
+
+DebugLevel Lwm2m_GetLogLevel()
+{
+    return debugLevel;
+}
+
+void Lwm2m_SetAwaLogLevel(AwaLogLevel level)
```

Comment:
Had to create a mapping between AwaLogLevel and DebugLevel - However we should really use a single enum for all logs. Libcoap uses a syslog style with 8 log levels, whereas AwaLogLevel only has 4. @chris-dewbery-imgtec @david-antliff-imgtec @seank-img